Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -49,7 +49,6 @@ final class PledgePaymentMethodsViewController: UIViewController {
       |> ksr_addSubviewToParent()
 
     self.tableView.registerCellClass(PledgePaymentMethodCell.self)
-    self.tableView.registerCellClass(PledgePaymentSheetPaymentMethodCell.self)
```

Comment:
n.B. this view controller still displays the list of payment methods in two sections, with the top section being 'new' cards and the bottom section being 'existing' cards. However, they use the same cell type and same backing data (a `UserCreditCards.CreditCard`).